app/controllers/application_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ApplicationController < ActionController::API
+
+private
+
+  def json_errors(errors, response_status)
+    render json: { data: { errors: errors, status: response_status } }, status: response_status
+  end
```

Comment:
Love how you placed this in the application controller to allow for inheritance!